LoopKitUI/DeviceManagerUI.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  DeviceManagerUI.swift
+//  LoopKitUI
+//
+//  Created by Rick Pasetto on 6/30/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+public protocol DeviceManagerUI {
+    /// An image representing the device configuration
+    static var image: UIImage? { get }
+    /// A localized name of the device to display to the user
+    static var name: String { get }
+    /// A localized detail description of the device to display to the user
+    static var details: String { get }
+}
+
+public extension DeviceManagerUI {
+    static var image: UIImage? { return nil }
+    static var name: String { return "" }
+    static var details: String { return "" }
+}
+
+public extension DeviceManagerUI {
+    var smallImage: UIImage? {
+        return type(of: self).image
+    }
+}
```

Comment:
I'm afraid this still doesn't cover the DIY use case—if the underlying requirement is static, MinimedPumpManager can't vend the colored version. Is there a reason the protocol property needs to be static?

Reply:
I believe the reason was so that the therapy settings flow could access the device manager images instead of having them be hard-coded

Reply:
The setup screen needs to use the *instantiated* version of the device manager images, because, as Michael indicated, the device manager does not necessarily vend a static image; it may change depending on the actual paired device.  Minimed pairs with different colored pumps, and vends appropriate images for the actual pump.  We could have different variants of other pumps and cgms, too, serviced by the same DeviceManager.

Reply:
The onboarding flow needs to be able to show images of available devices before they are set up (i.e. constructed).  Can MinimedPumpManager vend a generic colored version for static `image`, and override `smallImage`?

Reply:
Seems reasonable to me—note that `smallImage` will need to be made a protocol requirement in order to be overridden.

Reply:
@ps2 Does the latest code seem ok to you?  @mpangburn set me straight, making sure that the _instantiated_ version of `smallImage` is a protocol requirement, and that `DeviceManagerUI` is a refinement of `DeviceManager`.